-1 lessons away from enlightenment in python2 koans

Still have regex to do (maybe more) and this is what's displaying

...

Thinking TelevisionTest
  test_can_set_the_channel has expanded your awareness.
  test_edge_case_on_off has expanded your awareness.
  test_it_also_turns_off has expanded your awareness.
  test_it_turns_on has expanded your awareness.

Thinking AboutExtraCredit
  test_extra_credit_task has expanded your awareness.

Thinking AboutRegex
  test_matching_literal_text has damaged your karma.

You have not yet reached enlightenment ...
  AssertionError: I want my name

Please meditate on the following code:
  File "/home/tom/Dropbox/Misc Coding/python_koans/python2/koans/about_regex.py", line 27, in test_matching_literal_text


You have completed 297 koans and 39 lessons.
You are now 6 koans and -1 lessons away from reaching enlightenment.

Explicit is better than implicit.

--- a/python2/runner/sensei.py
+++ b/python2/runner/sensei.py
@@ -33,7 +33,7 @@ class Sensei(MockableTestResult):
                 self.stream.writeln()
                 self.stream.writeln("{0}{1}Thinking {2}".format(
                     Fore.RESET, Style.NORMAL, helper.cls_name(test)))
-                if helper.cls_name(test) != 'AboutAsserts':
+                if helper.cls_name(test) not in ['AboutAsserts', 'AboutExtraCredit']:
                     self.lesson_pass_count += 1
 
     def addSuccess(self, test):

--- a/python3/runner/sensei.py
+++ b/python3/runner/sensei.py
@@ -33,7 +33,7 @@ class Sensei(MockableTestResult):
                 self.stream.writeln()
                 self.stream.writeln("{0}{1}Thinking {2}".format(
                     Fore.RESET, Style.NORMAL, helper.cls_name(test)))
-                if helper.cls_name(test) != 'AboutAsserts':
+                if helper.cls_name(test) not in ['AboutAsserts', 'AboutExtraCredit']:
                     self.lesson_pass_count += 1
 
     def addSuccess(self, test):

